src/main/java/com/blibli/future/repository/EmployeeAbsencePermitRepository.java

```diff
@@ -11,14 +11,28 @@
 import com.blibli.future.enums.AbsencePermit;
 import com.blibli.future.model.Employee;
 import com.blibli.future.model.EmployeeAbsencePermit;
-import com.blibli.future.vo.SickEmployeeVo;
+import com.blibli.future.vo.SingleReportVo;
 
 @Repository
 public interface EmployeeAbsencePermitRepository extends JpaRepository<EmployeeAbsencePermit, String>{
 	public List<EmployeeAbsencePermit> findByEmployeeAndRequestDateBetween(Employee employee, LocalDate dateStart,LocalDate dateEnd);
 	public EmployeeAbsencePermit findOneById(String id);
-	@Query(value = "select new com.blibli.future.vo.SickEmployeeVo(e.nik, e.fullName, e.nameOfDept, count(e)) "
+	@Query(value = "select new com.blibli.future.vo.SingleReportVo(e.nik, e.fullName, e.nameOfDept, count(e)) "
 			+ "from EmployeeAbsencePermit eap JOIN Employee e ON eap.employee.nik = e.nik "
 			+ "WHERE e.nameOfDept = (?1) and eap.absencePermit = (?4) and (eap.requestDate between (?2) and (?3)) group by e.nik ORDER BY count(e) DESC")
-	public List<SickEmployeeVo> countSickEmployee(String deparment, LocalDate startDate, LocalDate endDate, AbsencePermit absencePermit, Pageable pageable);
+	public List<SingleReportVo> countSickEmployee(String deparment, LocalDate startDate, LocalDate endDate, AbsencePermit absencePermit, Pageable pageable);
```

Comment:
Just asking, `AbsencePermit` in the parameter is used for filtering what kind of absence, isn't it?
If that's the case this method is not only counting the "sick" employee right? Since we have other kind of absence permit.
But if this method is really intended to count just the sick employee, is it possible to put "sick" in the query and remove it from the parameter?